File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        for (int i = 0; i < numbers.size() - 1; i++) {
            int complement = target - numbers[i];
            for (int j = i; j < numbers.size(); j++) {
                if (numbers[j] == complement && numbers[j] != numbers[i]) {
                    return {i+1,j+1};
                }
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/merge-triplets-to-form-target/submission-0.py
class Solution:
    def mergeTriplets(self, triplets: List[List[int]], target: List[int]) -> bool:
        tx, ty, tz = target
        found_x = found_y = found_z = False

        for a, b, c in triplets:
            if a > tx or b > ty or c > tz:
                continue

            if a == tx:
                found_x = True
            if b == ty:
                found_y = True
            if c == tz:
                found_z = True

            if found_x and found_y and found_z:
                return True
            
        return found_x and found_y and found_z